src/test/java/co/rsk/bitcoinj/script/ScriptTest.java

```diff
@@ -262,8 +252,8 @@ public void createAndUpdateEmptyInputScript() {
     @Test
     public void testOp0() {
         // Check that OP_0 doesn't NPE and pushes an empty stack frame.
-        BtcTransaction tx = new BtcTransaction(PARAMS);
-        tx.addInput(new TransactionInput(PARAMS, tx, new byte[] {}));
+        BtcTransaction tx = new BtcTransaction(mainnetParams);
+        tx.addInput(new TransactionInput(mainnetParams, tx, new byte[] {}));
```

Comment:
WDYT if we put in a variable the value gets instantiating `TransactionInput` to get `Readiness`?

Reply:
Updated.